### DISCUSSION TITLE
Cadence target

### DISCUSSION THREAD

**Discussion by Lory3881** [dvmarinoff/Auuki#221] [Q&A] [answered]
Is that the chance to show both actual and target cadence ?

**Comment by dvmarinoff** [comment]
If I understand that right you'd like to be able to see both cadence and cadence target. If that's the case you can tap or click on the Cadence field and it will start to cycle between Cadence metrics. One of them shows the Cadence with the target next to it.
Here is how it looks:


https://github.com/user-attachments/assets/90250370-048b-4269-af45-ec3823b85dca

**Reply by Lory3881** [reply, marked as answer]
That's right!

**Reply by Lory3881** [reply]
Another question: whats the meaning of "simulation" in the setting tab and the difference between power or speed ?
![Uploading 20241212_083057.jpg…]()

**Reply by dvmarinoff** [reply]
It's about which metric should be used by the Simulation. Do you want speed to be calculated based on power + slope + coefficient of rolling resistance and other parameters, or do you want to just get the speed from a connected device like the Trainer or a Speed and Cadence sensor. Most trainers will report simple speed, which will be dependent on your gearing and won't be affected by the power output, but some trainers have build in simulation model and will report complex speed. So the choice is yours.

I've been thinking of adding a `?` section that shows on click or hover, and tries to explain that, but I am mostly short on time.

**Reply by Lory3881** [reply]
To see if I've understood Right: if I choose "simulation = speed" then the speed will be calculated by Auuki; other way, if I choose "simulation = power" if the power calculated by Auuki or what ?
What should I choose if I want both to be received by my elite zumo smar trainer? 
Thank you so much for quick reply!

**Reply by dvmarinoff** [reply]
Auuki will calculate the speed from power if you choose `power`. In your case if you want the speed as reported from the trainer you should select `speed`.

**Comment by Lory3881** [comment]
A suggestion: 
I want to use a dedicated thin stupid client for using auuki (still thinking if using it on remote or localhost) 24/7.

Do you think that this old setting can handle it?

Processor
Intel Atom Z3735F 1.33 GHZ
64 bit Quad-Core Processor
Memory
2 GB DDR3L 1333

**Reply by dvmarinoff** [reply]
If the system can handle a relatively recent version of a Blink based browser (Chrome/Edge) it should be fine.
The app itself consumes 4 to 10 MB of memory, the browser tab will need something like 50 +-10 MB.

Using the `auuki.com` version will give you access to syncing data from 3rd party platforms like Strava and Intervals.icu (This feature is almost ready).
The `localhost` version you'd be able to use without the need for an Internet connection.
